solr/contrib/ltr/README.md

```diff
@@ -391,12 +391,12 @@ About half the time for ranking is spent in the creation of weights for each fea
 <config>
   <!-- Query parser used to rerank top docs with a provided model -->
   <queryParser name="ltr" class="org.apache.solr.search.LTRQParserPlugin">
-     <int name="threadModule.maxThreads">10</int> <!-- Maximum threads to use for all queries -->
-     <int name="threadModule.maxQueryThreads">5</int> <!-- Maximum threads to use for a single query-->
+     <int name="threadModule.totalThreads">10</int> <!-- Maximum threads to use for all queries -->
+     <int name="threadModule.numThreads">5</int> <!-- Maximum threads to use for a single query-->
```

Comment:
I would argue this should be numPoolThreads or threadPoolSize or requestPoolSize or w/e . and numThreads should be numRequestThreads  or numThreadsPerRequest